docs/install.md

```diff
@@ -99,3 +102,6 @@ export PYTHONPATH=$PYTHONPATH:path-to-install-folder/python
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:path-to-install-folder/lib
 ```
+If using within an Anaconda environment:
```

Comment:
Might be good to make this a sub-section to capture the quirks that you have to deal with, especially for conda.